transport/src/main/java/io/netty/channel/nio/AbstractNioByteChannel.java

```diff
@@ -253,7 +251,6 @@ protected void doWrite(ChannelOutboundBuffer in) throws Exception {
                 if (done) {
                     in.remove();
                 } else {
-                    incompleteWrite(setOpWrite);
                     break;
```

Comment:
@Scottmitch no we want break here as we want to have it trigger incompleteWrite(...) outside of the for loop in https://github.com/netty/netty/pull/4216/files#diff-1b9567b1b2c19df64161d297624b02a0R261